.github/workflows/pr-verify.yml

```diff
@@ -6,6 +6,9 @@ on:
   pull_request:
     branches:
       - main
+  push :
+    branches:
+      - main
```

Comment:
re: my previous comment, it seems like this is limiting the entire pipeline to:
* PRs that target `main` as the destination branch
* Pushes directly to `main` or merges to `main`

So if I have a PR on a sub-branch, the pipeline won't run at all:
branches: `main/branch1` and `main/branch_1/branch_1a`
PR on `main/branch_1a` -> `main/branch_1` won't run CI in the PR itself because the target branch is not `main`
Merging the PR will not run CI because the branch being merged to is not `main`

Not a super big deal as we don't tend to do sub-branches, but food for thought.

Reply:
Yeah i mean its a simple pipelines that does the job for now